templates/hoover.nomad

```diff
@@ -192,6 +192,12 @@ job "hoover" {
         {% if config.snoop_pdf_preview_enabled %}
           SNOOP_PDF_PREVIEW_URL = "http://{% raw %}${attr.unique.network.ip-address}{% endraw %}:9990/_pdf-preview/"
         {% endif %}
+        {% if config.snoop_image_classification_object_detection_enabled %}
```

Comment:
This entire `env{}` block appears 3 times in this file. Let's move it into a macro at the top of this file called `snoop_dependency_envs()` and call it wherever necessary